### ISSUE TITLE
[P3.1][Sourcing][Accessibility][Pretest][Table sort]The screen reader does not output the sort direction because the column headings are not marked with aria-sort. 

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#10820]
#### Is this a bug, enhancement, or feature request?
Bug
#### Describe your proposal.
The screen reader does not output the sort direction because the column headings are not marked with aria-sort. 
expectation: The sort direction should be perceptible, e.g. per aria-sort

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)

Angular: 15
core/.platform: 0.43.13
#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
https://ng-15-downport--fundamental-ngx.netlify.app/#/platform/table/basic#p13-table-sort

**Comment by N1XUS** [comment]
@droshev `aria-sort` should be applied only to one `th` element per table. But Platform Table has the ability to set sorting with multiple columns

**Comment by nikolay-kolarov** [comment]
Hello Colleagues,

aria-sort is the right attribute to use. I checked that it is used already in UI5 when you have multiple sorted columns and it is correctly announced by Jaws. So let`s go for it and if we have complaints we can use the linked issue as a prove that it is in discussion in the ARIA group. 

Best Regards,
Nikolay
